app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,9 @@
 
   <body>
     <nav class = "topnav">
+      <%= link_to "Home Page", "/"%>
```

Comment:
If we have time lets be sure to refactor this into a drop down menu with css.  It will give it a cleaner user look and still be able to pass tests. https://www.w3schools.com/howto/howto_css_dropdown.asp